tests/neg/i18533.check

```diff
@@ -1,8 +1,8 @@
+-- Error: tests/neg/i18533/Non-SCALA_ONLY.java:4:7 ---------------------------------------------------------------------
```

Comment:
The order is important here (case sensitivity), since it's failing on Windows...